src/modules/menu/components/SimpleCategory.tsx

```diff
@@ -1,24 +1,83 @@
 import { type CategoryProps } from "@/modules/menu/components/Category";
-import { List } from "antd";
+import { type Food } from "@/modules/mock/foods";
+import { PlusOutlined } from "@ant-design/icons";
+import styled from "@emotion/styled";
+import { Button, Col, Image, List } from "antd";
 import React from "react";
 
 const SimpleCategory: React.FC<CategoryProps> = ({ category, foods }) => {
   return (
-    <List
+    <StyledList
       style={{
         scrollMarginTop: "112px", // very important for anchor to work
+        overflow: "hidden",
       }}
       id={category?.id}
       header={<div>{category?.name}</div>}
```

Comment:
The header of list is H4 According to the design in Figma.

Reply:
done krub